Modules/Write-ObjectToSQL/Write-ObjectToSQL.psm1

```diff
@@ -337,7 +337,18 @@ function Write-ObjectToSQL
                    ParameterSetName='mssql',
                    ValueFromPipeline=$False)]
         [ValidateNotNullorEmpty()]
-        [string]$PrimaryKey
+
+        [string]$PrimaryKey,
+
+        # Optional Switch to Drop Table before Data is loaded.
+        # This Switch is Usefull if you want to Drop your Table before. (No data Append)
+        [Parameter(Mandatory=$False,
+                   ParameterSetName='mssql',
+                   ValueFromPipeline=$False)]
+        [Parameter(Mandatory=$False,
+                   ParameterSetName='othersql',
+                   ValueFromPipeline=$False)]
+        [switch]$DropTable
```

Comment:
i've not tested it with othersql ... but mssql it's inserting/appending data without that switch.. but i do not want to insert, i want a fresh table everytime i call WriteSQL..